src/docfx/lib/path/PathUtility.cs

```diff
@@ -225,6 +223,28 @@ public static string UrlToShortName(string url)
             return result.ToString();
         }
 
+        private static bool IsNormalized(string path)
+        {
+            for (var i = 0; i < path.Length; i++)
+            {
+                var ch = path[i];
+                if (ch == '\\')
+                {
+                    return false;
```

Comment:
possible to return the last visited index as the start of the normalization part?